### PR TITLE
Lung cancer baseline screening incorporation

### PR DESCRIPTION
**Pull request by alibow**

### CODE REVIEW

File: docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst [hunk 1]
```diff
@@ -8,6 +8,41 @@ Tracheal, Bronchus, and Lung Cancer
    :local:
    :depth: 1
 
+.. list-table:: Abbreviations
```
Comment (aflaxman) [inline docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst:11]: Good addition!  Let's add this to our template.

**Review by aflaxman — APPROVED** [review]
I'm having trouble confirming that the ODF is going to do what it should in this, so let's be sure to review it together soon.

File: docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst (hunk 1 above)
Comment (alibow) [reply docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst:11]: Just made a PR!

**Comment by alibow** [comment]
> I'm having trouble confirming that the ODF is going to do what it should in this, so let's be sure to review it together soon.

Sounds good. Perhaps sometime today if you have time or during tomorrow's coworking session so we can do so before the meeting with Manoj?

**Comment by yongqx** [comment]
What baseline LDCT uptake rates you prefer to use for insured population? I found some studies in China reported as low as 1-2%.

**Comment by yongqx** [comment]
I am not sure whether our GBD forecast lung cancer incidence has included the baseline screening of LDCT, though we need verify against GBD data.

**Comment by alibow** [comment]
> I am not sure whether our GBD forecast lung cancer incidence has included the baseline screening of LDCT, though we need verify against GBD data.

My understanding through talking with @aflaxman is that we are going to assume that the forecasted rates are the expected rates with whatever we expect the baseline screening scenario coverage to be in the future.

**Review by yongqx — APPROVED** [review]

**Pull request merged**